test/json-schema.js

```diff
@@ -3,26 +3,57 @@ const fs = require('fs')
 const path = require('path')
 const validator = require('../')
 
-const schemaDraftDir = path.join(__dirname, '/json-schema-draft4')
-const files = fs
-  .readdirSync(schemaDraftDir)
-  .map(function(file) {
-    if (file === 'definitions.json') return null
-    if (file === 'refRemote.json') return null
-    if (file === 'ref.json') return null
-    const content = fs.readFileSync(path.join(schemaDraftDir, file))
-    return JSON.parse(content)
-  })
-  .filter(Boolean)
+const unsupported = new Set([
+  // Directories
+  'optional',
+  // Files
+  'definitions.json',
+  'refRemote.json',
+  'ref.json',
+  // Blocks
+  'format.json/validation of IP addresses',
+  'format.json/validation of IPv6 addresses',
+  'items.json/items and subitems',
+  // Specific tests
+  'items.json/an array of schemas for items/JavaScript pseudo-array is valid',
+  'maxLength.json/maxLength validation/two supplementary Unicode code points is long enough',
+  'minLength.json/minLength validation/one supplementary Unicode code point is not long enough',
```

Comment:
The two `maxLength/minLength` ones are fixed in https://github.com/ExodusMovement/is-my-json-valid/pull/22

Reply:
This one is merged now, so these tests can be reapplied

Reply:
Done, thanks! Rebased.